### PR TITLE
fix match multiple id error

### PR DESCRIPTION
**Pull request by nevermore3**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: tests/tck/features/match/MatchById.feature [hunk 1]
```diff
@@ -973,3 +973,68 @@ Feature: Match By Id
       | ("Warriors" :team{name: "Warriors"})                                                                        |
       | ("Hornets" :team{name: "Hornets"})                                                                          |
       | ("Spurs" :team{name: "Spurs"})                                                                              |
+
+  Scenario: match by multiple id
+    When executing query:
+      """
+      MATCH (a)-[e:like]->(b)
+      WHERE id(a) == 'Tim Duncan' OR id(b) == 'Tony Parker'
```
Comment (Shylock-Hg) [inline tests/tck/features/match/MatchById.feature:981]: This should report error

Comment (Shylock-Hg) [inline tests/tck/features/match/MatchById.feature:981]: Should add integer vid test case too.

Comment (nevermore3) [reply tests/tck/features/match/MatchById.feature:981]: use like index not vertexIDseek

Comment (Shylock-Hg) [reply tests/tck/features/match/MatchById.feature:981]: Emm. It's not related to this VIdSeek, better not put in this file.

Comment (nevermore3) [reply tests/tck/features/match/MatchById.feature:981]: ok， add case in next pr

Comment (Shylock-Hg) [reply tests/tck/features/match/MatchById.feature:981]: And you should add fail case of or expression.

Comment (Shylock-Hg) [reply tests/tck/features/match/MatchById.feature:981]: Why don't add in this PR directly?

Comment (nevermore3) [reply tests/tck/features/match/MatchById.feature:981]: @Sophie-Xie  waiting for this PR to publish test version

**Review by Shylock-Hg — APPROVED** [review]

File: tests/tck/features/match/MatchById.feature (hunk 1 above)
Comment (nevermore3) [reply tests/tck/features/match/MatchById.feature:981]: > Emm. It's not related to this VIdSeek, better not put in this file.

it's ok to put this case here

**Pull request merged**